src/Shinkansen.h

```diff
@@ -0,0 +1,33 @@
+#pragma once
+#include <stdio.h>
+#include <sys/socket.h>
+#include <unistd.h>
+#include <stdlib.h>
+#include <netinet/in.h>
+#include <string.h>
+#include <iostream>
+
+#include "Request.h"
+#include "Response.h"
+
+using namespace std;
+
+class Shinkansen
+{
+private:
```

Comment:
classes are private by default, no need to explicitly repeat it ... (although some people like to do it anyway)